unit-test

```diff
@@ -19,8 +19,8 @@ cmake -S . -B $build -DCMAKE_BUILD_TYPE=Debug -GNinja -DBUILD_P11=ON
 cmake --build $build --target aklite-tests
 
 echo "## Running clang tidy checks"
-clang_tests=$(ninja -C $build help | grep aktualizr_clang_tidy-src- | cut -d: -f1)
-cmake --build $build --target $clang_tests
+#clang_tests=$(ninja -C $build help | grep aktualizr_clang_tidy-src- | cut -d: -f1)
+#cmake --build $build --target $clang_tests
```

Comment:
do you re-enable this later on?

Reply:
No. There is a dedicated CI run in the github workflow `Clang Format And Tidy` to do `format` and `tidy` checks.

An engineer can/should run `make [-f dev-flow.mk] format | tidy` to do the checks on their host. It's more convenient to have dedicated targets in `makefile` for each type of checks (test, format, tidy, etc) and run a specific check/test at a specific time compared to running all checks/tests sequentially.